mabwiser/mab.py

```diff
@@ -812,12 +866,15 @@ def __init__(self,
             lp = _ThompsonSampling(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.binarizer)
         elif isinstance(learning_policy, LearningPolicy.UCB1):
             lp = _UCB1(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.alpha)
+        elif isinstance(learning_policy, LearningPolicy.LinGreedy):
+            lp = _Linear(self._rng, self.arms, self.n_jobs, self.backend, learning_policy.l2_lambda,
```

Comment:
So `alpha` is explicitly set to zero for LinGreedy?

Reply:
It is set to zero, although alpha is not used in LinGreedy anywhere.